doc/input-output-reference/src/overview/group-unitary-equipment.tex

```diff
@@ -2843,6 +2843,12 @@ \subsection{AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass}\label{airloophvacun
 \caption{Schematic of a CBVAV unitary system (blow through fan placement) \protect \label{fig:schematic-of-a-cbvav-unitary-system-blow}}
 \end{figure}
 
+\begin{figure}[hbtp]
+\centering
+\includegraphics[width=0.9\textwidth, height=0.9\textheight, keepaspectratio=true]{media/ChangeoverBypassVAV-ReturnPlenumOrMixer.png}
```

Comment:
I think this is causing the "build failure" on Linux.  The filename is subtly different:

```
media/ChangeoverBypassVAV-ReturnPlenumOrMixer.png    -- in this spot
media/ChangeoverBypassVAV-ReturnPlenumOrMixer.PNG   -- the actual file name
```

Keeping in mind that Linux is case sensitive.  Also, make sure this new image is added to the CMake rules [here](https://github.com/NREL/EnergyPlus/blob/5e66d36f7c13d2f0f794370eabd85f0dd90db55e/doc/input-output-reference/CMakeLists.txt#L117) so that if the image is changed, CMake picks up that change and will rebuild the doc, otherwise it doesn't know that the image is a dependency of this build.